src/components/Button/Button.tsx

```diff
@@ -80,7 +80,6 @@ function Button(props: React.PropsWithChildren<ButtonProps>) {
   }
 
   styles = useStyleConfig("Button", { variant });
-
```

Comment:
Curious, was this your editor?

Reply:
I probably put a console.log here to see what was going on and then took it out and it registered as changed.